src/routes/Project/containers/ProjectParamsContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import ProjectParams from '../components/ProjectParams';
+import { updateTempo, updateTimeSig, updateTimeDiv } from '../actions/TableActions';
+
+const mapDispatchToProps = {
+  updateTempo,
+  updateTimeSig,
+  updateTimeDiv
```

Comment:
Just keep the state info in the parent component http://stackoverflow.com/questions/24147331/react-the-right-way-to-pass-form-element-state-to-sibling-parent-elements

Alternatively we could have separate reducers for local state changes